getting_started/create_vm/destroy_vm.sh

```diff
@@ -2,16 +2,21 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the Apache 2.0 License.
 
-set -ex
+set -e
 
-read -p "Are you sure? " -n 1 -r
+if [ "$#" -ne 1 ]; then
+  echo "make_vm.sh takes an Azure subscription name"
```

Comment:
This says `make_vm.sh`, but is in `destroy_vm.sh`. Think this is easier to read as a usage hint, ie:

```
echo "Usage: $0 SUBSCRIPTION_NAME"
```

Reply:
yes